Denysyefimov

Detroiman opened the pull request:
<h1>Description</h1>

*Please include summary of the change in english*


<h2>How has this been tested?</h2>

- [ ] Executed code few times from start to end without crashing
- [ ] Only ran once
- [ ] Code have dedicated unit tests

Detroiman committed:
Add files via upload

Detroiman committed:
Delete denysyefimov_task1 directory

Detroiman committed:
Add files via upload

meanalexrin commented on denysyefimov_task1/denysyefimov_houseprices.py:
```diff
@@ -0,0 +1,170 @@
+
+if __name__ == '__main__':
+    # add data in file
+    train = pd.read_csv('C:/Users/Gyrocopter_UA/PycharmProjects/pythonProject/data/train.csv')
```
Just for the future:
try not to include paths, IPs, personal details on any other personalized or secret data into github commits
This kind of data may unintentionally leak some secret information about a business if you working in a commercial project

meanalexrin commented on denysyefimov_task1/denysyefimov_houseprices.py:
```diff
@@ -0,0 +1,170 @@
+    deviation = np.log(train.SalePrice)
+    print("Deviation: ", deviation)
+    plt.hist(deviation, color='black')
+    #plt.show()
```
Consider removing commented/unused code
And try to separate drawing from main logic - currently there are a lot of calls to `plt` that rendered but newer shown

meanalexrin reviewed (approved):
Please move your code to 
`denysyefimov_code/task...`
And please fix flake warnings 

Besides this - Looks Good To Me :+1:

meanalexrin reviewed (changes requested):
Nah, I was going to request changes but accidentally approved in previous comment
Please move code and fix warnings as it was mentioned in previous comment

Detroiman committed:
Delete denysyefimov_task1 directory

Detroiman committed:
Add files via upload

Removed all references to my local directory and removed code not used in .py extension

Detroiman committed:
Add files via upload

Removed all references to my local directory and removed code not used in .py extension

Detroiman committed:
Delete Classification_example_with_Iris_dataset.py

Detroiman committed:
Delete README.md

Detroiman committed:
Delete Classification_example_with_Iris_dataset.ipynb

Detroiman committed:
Add files via upload

Detroiman committed:
Delete tasks/task_1 directory

Yullia reviewed (changes requested):
Please, fix flake warnings and errors

Detroiman committed:
Update denysyefimov_houseprices.py

Detroiman committed:
Update denysyefimov_houseprices.py

Detroiman committed:
Delete denysyefimov_code directory

Detroiman committed:
Add files via upload

Detroiman committed:
Delete denysyefimov_houseprices.ipynb

Detroiman committed:
Delete denysyefimov_titanic.ipynb

Detroiman committed:
Delete denysyefimov_code directory

Detroiman committed:
Add files via upload

Detroiman committed:
Update denysyefimov_face_recogn.py

Detroiman committed:
Update denysyefimov_face_recogn.py

Detroiman committed:
Update denysyefimov_face_mask.py
